src/styles/watch.scss

```diff
@@ -0,0 +1,51 @@
+.watch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  height: 420px;
+  width: 420px;
+  border-radius: 50%;
+  border: 4px solid $watch-border-color;
+
+  &__second-hand {
+    position: absolute;
+    bottom: 50%;
+
+    &--style {
+      width: 5px;
+      height: 200px;
+      border-radius: 50%;
+      background-color: $watch-hand-color;
+
+      animation: clock 60s steps(60) infinite;
+      transform-origin: bottom;
+    }
+  }
+
+  &__minute-hand {
+    position: absolute;
+    bottom: 50%;
```

Comment:
don't repeat the styles. you can use extend, or make common class for both hands with 2 modifiers